gap/PGMatrix.gd

```diff
@@ -0,0 +1,96 @@
+#!
+#!
+#! @Chapter Hyperbolic lattice symmetries
```

Comment:
I am wondering a bit about terminology and scope here.
1. What is this chapter really about? "Hyperbolic lattice symmetries" seems to generic to me, since the whole package is about that. After all, the hyperbolic lattice symmetries are the elements of the triangle groups... We need a more specific/accurate title for this chapter.
2. Is it maybe mostly about point-group symmetries? The section below is about how point-group symmetries act on momenta.
3. It could also be a chapter about action on momenta, but does it make sense to include other symmetry elements as well (which would mean translations, that seems trivial - I cannot imagine a nontrivial action of translations on momenta).